Description: vmstat.8: --unit changes si/so
 The --unit option said si/so were not changed.
 The si/so description said they were.
 The code, which is the final decision, says yes.
 .
 Update vmstat.8 to remove note that si/so not changed by --unit
Author: Craig Small <csmall@debian.org>
Origin: upstream, https://gitlab.com/procps-ng/procps/-/commit/43ece4d2b0b8d5163ed118e76eff98c4988620db
Bug-Debian: https://bugs.debian.org/1061944
Applied-Upstream: 4.0.5
Reviewed-by: Craig Small <csmall@debian.org>
Last-Update: 2025-04-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/man/vmstat.8
+++ b/man/vmstat.8
@@ -1,5 +1,5 @@
 .\"
-.\" Copyright (c) 2002-2023 Craig Small <csmall@dropbear.xyz>
+.\" Copyright (c) 2002-2024 Craig Small <csmall@dropbear.xyz>
 .\" Copyright (c) 2012-2023 Jim Warner <james.warner@comcast.net>
 .\" Copyright (c) 2011-2012 Sami Kerola <kerolasa@iki.fi>
 .\" Copyright (c) 1994      Henry Ware <al172@yfn.ysu.edu>
@@ -10,7 +10,7 @@
 .\" (at your option) any later version.
 .\"
 .\"
-.TH VMSTAT 8 "2023-01-18" "procps-ng" "System Administration"
+.TH VMSTAT 8 "2024-07-19" "procps-ng" "System Administration"
 .SH NAME
 vmstat \- Report virtual memory statistics
 .SH SYNOPSIS
@@ -81,8 +81,7 @@
 .RI ( m ),
 or 1048576
 .RI ( M )
-bytes.  Note this does not change the swap (si/so) or block (bi/bo)
-fields.
+bytes.  Note this does not change the block (bi/bo) fields.
 .TP
 \fB\-t\fR, \fB\-\-timestamp\fR
 Append timestamp to each line
